qiita_pet/util.py

```diff
@@ -51,8 +56,13 @@ def clean_str(item):
     -------
     str
         cleaned string
+
+    Notes
+    -----
+    This function removes colons and periods from strings, and replaces spaces
+    with underscores.
     """
-    return str(item).replace(" ", "_").replace(":", "")
+    return re.sub(clean_str_regex, "", str(item).replace(" ", "_"))
```

Comment:
For what this is being used for, stripping spaces out completely might work fine as well. Testing now.